plugins/ckeditor5-woltlab-attachment/src/woltlabattachment.ts

```diff
@@ -57,7 +60,9 @@ export class WoltlabAttachment extends Plugin {
 
               const viewWriter = conversionApi.writer;
               let img = conversionApi.mapper.toViewElement(data.item);
+              let figure = null;
```

Comment:
```suggestion
              let figure = undefined;
```

`null` is super wonky in JavaScript and whenever possible (and not mandated by an existing API) you should use `undefined` instead. `undefined` means that there is no value while `null` is typically used to signal a negative match.